tests/test_serializers.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime
+
+from api.serializers import FeedsResponseSerializer, FeedsSerializer
+from django.test import TestCase
+from greedybear.consts import PAYLOAD_REQUEST, SCANNER
+from greedybear.models import IOC
+from rest_framework.serializers import ValidationError
+
+
+class FeedsSerializersTestCase(TestCase):
+    def test_valid_fields(self):
+        data_ = {"feed_type": "all", "attack_type": "all", "age": "recent", "format": "txt"}
+        serializer = FeedsSerializer(data=data_)
+        valid = serializer.is_valid(raise_exception=True)
+        self.assertEqual(valid, True)
```

Comment:
This is correct but I'd like to test all the permutations: We could use an array for each filed and nest them in some for loop to test all the permutations